styles/utilities/_utilities.scss

```diff
@@ -160,6 +160,12 @@
   max-width: 100% !important;
 }
 
+/// Changes the width of the element to `auto`.
+/// @group width
+.sprk-u-Width--auto {
```

Comment:
Why did we add this one? I probably knew at one time but forgot.

Reply:
I don't remember. It was a support room request at some point I think. We talked about it like, 6 months ago. Ha.